paper_framework_sources/p2079_system_execution_context.bs

```diff
@@ -372,10 +353,10 @@ public:
 };
 </pre>
 
- - `get_system_scheduler()` returns a scheduler that provides a view on some underlying execution context supporting *parallel forward progress*, with at least one thread of execution (which may be the main thread).
- - two objects returned by `get_system_scheduler()` may share the same execution context.
+ - `get_parallel_scheduler()` returns a scheduler that provides a view on some underlying execution context supporting *parallel forward progress*, with at least one thread of execution (which may be the main thread).
+ - two objects returned by `get_parallel_scheduler()` share the same execution context.
```

Comment:
Is requiring this intended to avoid bad QOI implementations?

Reply:
It was more a clarification of what to expect. But, thank you for pointing that out, this doesn't always hold, so I had to relax it a bit.